magento/pwa-studio
# [feat]: Checkout Page Order Summary

## Description

Adds the order summary to the checkout page. I reused the price summary component from cart page so we can discuss how to properly store/structure the directories here if we want.

That said, in re-using the pre-existing component I was able to skip over a lot of repetition. The only difference between cart and checkout summaries are that on cart we have the word "Estimated" in front of shipping, tax and total as well as a "proceed to checkout" button. On the checkout page there is no button and we remove the word "Estimated".

I also added a Link functionality to move us from cart to checkout by clicking the button :)

## Related Issue





Closes [PWA-184](https://jira.corp.magento.com/browse/PWA-184).

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Add an item to your cart.
2. Go to the cart page, estimate your shipping, add some other things like coupon/gift card. Note how the price summary looks as well as the totals therein.
3. Click to proceed to checkout. The totals should mirror what you saw on cart. The text should not say "Estimated" and there should be no "Proceed to checkout" button.
4. Regress in mobile.

## Screenshots / Screen Captures (if appropriate)
**Video**
[![Image from Gyazo](https://i.gyazo.com/9a5345f1a418c50f50d082ba88ef816b.gif)](https://gyazo.com/9a5345f1a418c50f50d082ba88ef816b)

**Desktop**
[![Image from Gyazo](https://i.gyazo.com/ae53f46b19b8b07c98def2afcab8bbe4.png)](https://gyazo.com/ae53f46b19b8b07c98def2afcab8bbe4)

**Mobile**
[![Image from Gyazo](https://i.gyazo.com/8a7ab8a8c838f5f2072eaf92bc51e92d.png)](https://gyazo.com/8a7ab8a8c838f5f2072eaf92bc51e92d)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
